packages/e2e-tests/src/jest.config.js

```diff
@@ -37,7 +37,7 @@ export default {
   testMatch: ['**/specs/**/*.[jt]s', '**/?(*.)spec.[jt]s'],
   // @jest/test-sequencer is the default.
   testSequencer:
-    'true' === process.env.CI
+    'true' === process.env.NODE_ENV
```

Comment:
Same here? I guess this one doesn't matter since process should be available when running jest.

Reply:
Yes, same here. This code is executed in Node context, not browser context. So using `process.env` is correct. 

But it really should stay `process.env.CI`